test/CMakeLists.txt

```diff
@@ -47,7 +64,7 @@ if (WITH_TEST_GENERATORS)
   # and make an executable target for it named 'generator_foo'
   file(GLOB TESTS RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}/generator" "${CMAKE_CURRENT_SOURCE_DIR}/generator/*_jittest.cpp")
   foreach(test ${TESTS})
-    string(REPLACE "_jittest.cpp" "" name "generator_jit_${test}")
+    string(REPLACE "_jittest.cpp" "" name "generator_exec_jit_test_${test}")
```

Comment:
Perhaps, but a default windows 8 install has this problem, and I don't want
to tell users to change their file system configuration to build Halide :)

On Thu, Aug 13, 2015 at 11:55 AM, Steven Johnson <notifications@github.com>
wrote:

> In test/CMakeLists.txt
> <https://github.com/halide/Halide/pull/892#discussion_r37011619>:
>
> > @@ -47,7 +64,7 @@ if (WITH_TEST_GENERATORS)
> >    # and make an executable target for it named 'generator_foo'
> >    file(GLOB TESTS RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}/generator" "${CMAKE_CURRENT_SOURCE_DIR}/generator/*_jittest.cpp")
> >    foreach(test ${TESTS})
> > -    string(REPLACE "_jittest.cpp" "" name "generator_jit_${test}")
> > +    string(REPLACE "_jittest.cpp" "" name "generator_exec_jit_test_${test}")
>
> This is a dumb response, but...
>
> Surely Windows systems in 2015 can be configured to avoid this Neanderthal
> limitation?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/halide/Halide/pull/892/files#r37011619>.
>